client/src/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Image from './image.jsx';
+import Description from './description.jsx';
+
+const flexStyle = {
+  display: 'flex'
+};
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.state.book = {};
+  }
+
+  componentDidMount() {
+    const setBook = this.updateState.bind(this);
+
+    fetch('http://localhost:3004/books/7')
+      .then(function(response) {
+        return response.json();
+      })
+      .then(function(myJson) {
+        setBook(myJson);
```

Comment:
better use `arrow callback functions` (`Airbnb-recommended`) & this'll enable you to invoke `this.setState` here without using another function let alone having to bind it.